GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Aero.cfg

```diff
@@ -0,0 +1,184 @@
+@PART[M2X_Tailboom|M2X_InlineIntake|M2X_AeroIntake|M2X_circularintake|M2X_EngineShroud|M2X_MantaIntake|M2X_Precooler|M2X_Shockcone]:FOR[RealismOverhaul]
+{
+	%skinMaxTemp = 2500
+}
+
+@PART[M2X_Tailboom]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.915
+}
+
+@PART[M2X_InlineIntake]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+	@mass = 0.875
+    rescaleCube = 1
+	@DRAG_CUBE
+	{
+		rescaleX = 1.722222
+		rescaleY = 1.722222
+		rescaleZ = 1.722222
+	}
+	
+	MODULE
+    {
+        name = ModuleFuelTanks
+        volume = 7500
+        type = Fuselage
+    }
+
+	@MODULE[ModuleResourceIntake]
+	{
+		@name = AJEInlet
+		Area = 1.5
+		#@AJE_TPR_CURVE_DEFAULTS/PitotTube/TPRCurve {}
+
+		inletTitle = #$@AJE_TPR_CURVE_DEFAULTS/PitotTube/title$
+		inletDescription = #$@AJE_TPR_CURVE_DEFAULTS/PitotTube/description$
+	}
+}
+
+@PART[M2X_AeroIntake]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+    @mass = 0.5
+    rescaleCube = 1
+	@DRAG_CUBE
+	{
+		rescaleX = 1.722222
+		rescaleY = 1.722222
+		rescaleZ = 1.722222
+	}
+}
+@PART[M2X_circularintake]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    @mass = 0.275
+    rescaleCube = 1
+	@DRAG_CUBE
+	{
+		rescaleX = 1.722222
+		rescaleY = 1.722222
+		rescaleZ = 1.722222
+	}
+
+	@MODULE[ModuleResourceIntake]
+	{
+		@name = AJEInlet
+		Area = 1.2
+		#@AJE_TPR_CURVE_DEFAULTS/PitotTube/TPRCurve {}
+
+		inletTitle = #$@AJE_TPR_CURVE_DEFAULTS/PitotTube/title$
+		inletDescription = #$@AJE_TPR_CURVE_DEFAULTS/PitotTube/description$
+	}
+}
+
+@PART[M2X_EngineShroud]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.25
+}
+
+@PART[M2X_MantaIntake]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    @mass = 0.375
+    rescaleCube = 1
+	@DRAG_CUBE
+	{
+		rescaleX = 1.722222
+		rescaleY = 1.722222
+		rescaleZ = 1.722222
+	}
+
+	@MODULE[ModuleResourceIntake]
+	{
+		@name = AJEInlet
+		Area = 2
+		#@AJE_TPR_CURVE_DEFAULTS/AdjustableSupersonic/TPRCurve {}
+
+		inletTitle = #$@AJE_TPR_CURVE_DEFAULTS/AdjustableSupersonic/title$
+		inletDescription = #$@AJE_TPR_CURVE_DEFAULTS/AdjustableSupersonic/description$
+	}
+}
+
+@PART[M2X_Precooler]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    @mass = 1.22
+    rescaleCube = 1
+	@DRAG_CUBE
+	{
+		rescaleX = 1.722222
+		rescaleY = 1.722222
+		rescaleZ = 1.722222
+	}
+
+    MODULE
+    {
+        name = ModuleFuelTanks
+        volume = 1500
+        type = Fuselage
+    }
+
+	!MODULE[ModuleResourceIntake] {}
+	!RESOURCE[IntakeAir] {}
+
+	MODULE
```

Comment:
This seems incredibly powerful compared to other radiators ingame. Also, precoolers don't do anything with AJE.